data.sql

```diff
@@ -12,3 +12,24 @@ VALUES ('Pikachu', '2021-01-07', 15.04, false, 1);
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, 
 escape_attempts)
 VALUES ('Devimon', '2017-05-12', 11.00, true, 5);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES('Charmander', '2020-02-08', 0, false, 11.00);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES('Plantmon', '2021-11-15', 2, true, 5.7);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES('Squirtle', '1993-04-02', 3, false, 12.13);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES('Angemon', '2005-06-12', 1, true, 45.00);
```

Comment:
- [ ] I know it looks like a mistake, but here, the weight of all these animals is supposed to be negative. If this is the only issue, your project will be approved but make sure to apply these changes for the next milestone, if not, please make sure to read the rest of the comments.

Reply:
Thank You for pointing that out. It was a mistake and I will apply the changes.